packagehandlers/packagehandlers_test.go

```diff
@@ -2,7 +2,7 @@ package packagehandlers
 
 import (
 	"fmt"
-	testdatautils "github.com/jfrog/build-info-go/build/testdata"
+	testdatautils "github.com/jfrog/build-info-go/tests"
```

Comment:
I think we can remove the alias and just use "tests"
```suggestion
	"github.com/jfrog/build-info-go/tests"
```